samples/public/BlankUwpNet9App/BlankUwpNet9App.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net9.0-windows10.0.26100.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <WindowsSdkPackageVersion>10.0.26100.57</WindowsSdkPackageVersion>
+    <Platforms>x86;x64;arm64</Platforms>
+    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+    <PublishProfile>win-$(Platform).pubxml</PublishProfile>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <PublishAot>true</PublishAot>
+    <UseUwp>true</UseUwp>
+    <DisableRuntimeMarshalling>true</DisableRuntimeMarshalling>
+    <EnableMsixTooling>true</EnableMsixTooling>
+	  <UnitTestPlatformVersion Condition="'$(UnitTestPlatformVersion)' == ''">$(VisualStudioVersion)</UnitTestPlatformVersion>
+
+    <!-- Set this property to override the version of the MSIX package to use -->
+    <MicrosoftWindowsSDKBuildToolsMSIXPackageVersion>1.0.20241009.2</MicrosoftWindowsSDKBuildToolsMSIXPackageVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+	  <ProjectCapability Include="TestContainer" />
```

Comment:
Hmm, why is this not flowing through package props/targets 😕

Reply:
This is the way Sergio had set it up. Will follow up with him to understand.

https://devblogs.microsoft.com/devops/the-testcontainer-capability/

Reply:
Keeping this open as we can probably remove it but let's wait to have something working first.

Reply:
I see now this is set by `Microsoft.NET.Test.Sdk` which you don't reference.